rootfs/registry/dockerclient.py

```diff
@@ -46,7 +46,6 @@ def publish_release(self, source, config, target, deis_registry):
         self.tag(image, src_name, tag=src_tag)
 
         # build a Docker image that adds a "last-mile" layer of environment
-        config.update({'DEIS_APP': name, 'DEIS_RELEASE': tag})
```

Comment:
I think we'll need to inject these into the environment on `create` [here](https://github.com/deis/workflow/blob/15df775432a4d7f87c8d69805e7bab6ce857b838/rootfs/api/models.py#L221) for DEIS_APP and [here](https://github.com/deis/workflow/blob/15df775432a4d7f87c8d69805e7bab6ce857b838/rootfs/api/models.py#L485) for DEIS_RELEASE.